recipes/rabbitmq/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "rabbitmq-server" %}
+{% set version = "3.6.9" %}
+{% set sha256 = "012e8efbd0f40c086030803c92fea2e3af37b8d1515a553896fb5c1f3960bc40" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #path: ./rabbitmq_server-3.6.9
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://www.rabbitmq.com/releases/{{ name }}/v{{ version }}/{{ name }}-generic-unix-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - 3a8d7ed.patch  # merges rabbitmq/rabbitmq-server#1192
+    - a2f5620.patch  # wrapper to fake RABBITMQ_HOME location
+    - f52bca4.patch  # SYS_PREFIX is set to Conda's PREFIX
```

Comment:
:+1: for the comments. (Since the names are pretty enigmatic :stuck_out_tongue_closed_eyes:)